index.html

```diff
@@ -0,0 +1,638 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="#">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+    <title>The next web page</title>
+  </head>
+  <body>
+    <nav>
+      <div class="TNWWhiteLogo">
+        <img src="Images/tnw-white.svg" alt="">
+      </div>
+      <div class="divLinks">
+        <a href="#">LATEST</a>
+        <a href="#">HARD FORK</a>
+        <a href="#">PLUGGED</a>
+        <a href="#">READ ME</a>
+        <a href="#">GROWTH QUARTERS</a>
+        <a class="hideTablet" href="#">SHIFT</a>
+        <a class="hideTablet" href="#">NEURAL</a>
+      </div>
+      <div class="divIcon">
+        <a class="iconsSearch" href="#"><i class="fas fa-search"></i></a>
+        <a class="justMobile"href="#"><i class="fas fa-bars"></i></a>
+      </div>
```

Comment:
## Add space
- Add and space line 26
` <a class="justMobile" (here) href="#"><i class="fas fa-bars"></i></a>`